go.mod

```diff
@@ -2,6 +2,8 @@ module istio.io/istio
 
 go 1.24.0
 
+replace github.com/istio.io/api => github.com/tjons/api v0.0.0-20250504182006-697b066506ea
```

Comment:
Probably safe to remove this if you're ready for review.

Reply:
This is pinned until the API changes are approved and then I'll remove it. If I remove it right now the test suite will fail.